conf/openmetadata.yaml

```diff
@@ -120,12 +120,12 @@ logging:
 
 database:
   # the name of the JDBC driver, mysql in our case
-  driverClass: ${DB_DRIVER_CLASS:-com.mysql.cj.jdbc.Driver}
+  driverClass: ${DB_DRIVER_CLASS:-org.postgresql.Driver}
```

Comment:
why are we changing these defaults @mohityadav766

Reply:
my bad , was testing It on mysql and Postgres got pushed by mistake